src/components/Tabset/__test__/tabset.spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Tabset from '../index';
+import Tab from '../../Tab';
+
+const registerTabMockFn = jest.fn();
+
+describe('<Tabset />', () => {
+    it('should render the children passed', () => {
```

Comment:
you don't need to pass a Tab to test this instead you could pass anything like some string